390/HALEY_SCHMIDT.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "MP2"
+author: "Haley Schmidt"
+date: "9/16/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(readr)
+scbi_stem1 <- read_csv("tree_main_census/data/census-csv-files/scbi.stem1.csv")
+scbi_stem2 <- read_csv("tree_main_census/data/census-csv-files/scbi.stem2.csv")
+scbi_stem3 <- read_csv("tree_main_census/data/census-csv-files/scbi.stem3.csv")
+```
+
+```{r}
+library(ggplot2)
+library(tidyr)
+library(dplyr)
+```
+
+```{r}
+stem1 <- scbi_stem1 %>%
+  filter(dbh != "NULL") %>%
+  mutate(dbh = as.numeric(dbh)) %>%
+  select(sp, dbh, CensusID) %>%
+  group_by(sp) %>% 
+  summarize(avg_dbh_sp = mean(dbh),
+            census_year = mean(CensusID))
+
+stem2 <- scbi_stem2 %>%
+  filter(dbh != "NULL") %>%
+  mutate(dbh = as.numeric(dbh)) %>%
+  select(sp, CensusID, dbh) %>%
+  group_by(sp) %>% 
+  summarize(avg_dbh_sp = mean(dbh),
+            census_year = mean(CensusID))
+
+stem3 <- scbi_stem3 %>%
+  filter(dbh != "NULL") %>%
+  mutate(dbh = as.numeric(dbh)) %>%
+  select(sp, CensusID, dbh) %>%
+  group_by(sp) %>% 
+  summarize(avg_dbh_sp = mean(dbh),
+            census_year = mean(CensusID))
+
+stem_joined <- rbind(stem1, stem2, stem3)
+
+genus_dbh <- stem_joined %>%
```

Comment:
Hey Haley, here is a code review! A heads up that `scbi_spptable` is missing. You'll need to load that to ensure the .Rmd is completely reproducible

Reply:
Could you fix this and push the changes?

Reply:
Did you remember to push the change? b/c the code that loads `scbi_spptable` isn't there, and hence the file won't knit. It's a good idea to always knit once before pushing changes to make sure things look good.

Reply:
I tried again but I'm not sure if it worked, maybe we can look at this after class?

Reply:
I was able to knit your file, so it worked! One way you can check if your pushes were made is to look at the "Files Changed" tab on the top right of this page (The tabs are Conversation, Commits, Checks, Files Changes). That shows the "diff" = difference between what's in `master` vs your `HaleyMP2` branch